tools/fp20compiler/rc1.0_register.h

```diff
@@ -75,7 +75,7 @@ static const char* GetRegisterNameString(unsigned int reg_name) {
     case REG_DISCARD:
         return "0x0";
     case REG_ONE:
-        return "0x12";
+        assert(false);

```

Comment:
This should get a comment which explains that it's a pseudo-register which translates to `REG_ZERO`.

Reply:
I'm not happy with my comment. 😨I would love the comment to be a bit more explanation for the _why_ and not just _bla bla can't reach_. But its a complex topic(a.l. for me) and I also don't want to spam a multiline comment.